lib/livebook/session.ex

```diff
@@ -700,6 +717,7 @@ defmodule Livebook.Session do
 
   def handle_cast({:disconnect_runtime, client_pid}, state) do
     Runtime.disconnect(state.data.runtime)
+    send(self(), :system_memory)
```

Comment:
Instead of scheduling it here, what do you think about computing the node memory whenever we receive the `memory` updates from the evaluator?

Reply:
I didn't get that one either!
We won't have a runtime here anymore. So I go back to only tracking the system memory and directly in the session.